api/src/main/java/org/ccci/idm/user/User.java

```diff
@@ -411,12 +412,15 @@ public void setCruProxyAddresses(Collection<String> cruProxyAddresses) {
         this.cruProxyAddresses = cruProxyAddresses;
     }
 
-    public Collection<String> getCruPasswordHistory() {
-        return cruPasswordHistory;
+    public Set<String> getCruPasswordHistory() {
+        return ImmutableSet.copyOf(cruPasswordHistory);
```

Comment:
I don't quite understand how this solves the `List.subList(...)` serialization issue since in either the case underlying implementation was a `Set`. And, a `List` implementation is passed https://github.com/GlobalTechnology/idm-user-management/blob/master/api/src/main/java/org/ccci/idm/user/util/PasswordHistoryManager.java#L49 either way.